assets/styles/main.css

```diff
@@ -0,0 +1,1699 @@
+/* CSS RESET */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+}
+
+body {
+  min-height: 100vh;
+  scroll-behavior: smooth;
+  text-rendering: optimizeSpeed;
+  line-height: 1.5;
+}
+
+img {
+  max-width: 100%;
+  display: block;
+  object-fit: cover;
+}
+
+/* HEADER SECTION */
+
+.navcontainer {
+  position: relative;
+  width: 1250px;
+  margin: 0 auto;
+}
```

Comment:
- [ ] Bootstrap has a class for  `container` and `container-fluid`

- [ ] Bootstrap has class for `position-relative`